src/main/resources/db/migration/V1_49_1__contract_type_data.sql

```diff
@@ -0,0 +1,20 @@
+insert into contract_type(id, name)
+Values  ('72e60faf-b8e5-4699-9d7c-aef631cca71b', 'Accomodation'),
+        ('b402486d-991e-4977-9291-073a3526d60f', 'Education, Training and Employment'),
+        ('bf68c1c6-f83b-45ed-a6f7-495456e5745a', 'Finance, Debt and Benefits'),
+        ('e46e76e1-38da-46a4-afc0-d5e7283ebb03', 'Dependency and recovery'),
+        ('f9b59d2c-c60b-4eb0-8469-04c975d2e2ee', 'Personal Wellbeing'),
+        ('ab0ddafa-619a-47c1-89ca-3709b2833761', 'Lifestyle, associates and social inclusion'),
+        ('08fb8af5-b8d9-47cf-981a-90adaffd0c20', 'Structured Interventions');
```

Comment:
i think we should probably remove this line for right now until we really know what's going on with SIs